test_runner/regress/test_tenant_delete.py

```diff
@@ -556,6 +556,216 @@ def hit_run_failpoint():
     assert ps_http.get_metric_value("pageserver_tenant_manager_slots") == 0
 
 
+def test_tenant_delete_races_timeline_creation_01(
```

Comment:
Next time, this can be done with the pytest parametrization like:

```python
@pytest.mark.parametrize("nth", range(30))
def foobar(neon_env_builder: NeonEnvBuilder, pg_bin: PgBin, _nth: int):
  ...
```